static/style.css

```diff
@@ -89,6 +89,16 @@ body {
     background:linear-gradient(rgba(0, 0, 0, 0.294),rgba(255, 255, 255, 0.437)) ;
 }
 
+.source-heading {
```

Comment:
make github icon little bit left side like not touching the right side of window